test/Microsoft.Dnx.Tooling.FunctionalTests/DnuPublishTests2.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using System.Runtime.Versioning;
+using Microsoft.Dnx.Runtime;
+using Microsoft.Dnx.Runtime.Helpers;
+using Microsoft.Dnx.Testing;
+using Microsoft.Dnx.Tooling.Publish;
+using Newtonsoft.Json.Linq;
+using Xunit;
+
+namespace Microsoft.Dnx.Tooling.FunctionalTests
+{
+    [Collection(nameof(ToolingFunctionalTestCollection))]
+    public class DnuPublishTests2 : DnxSdkFunctionalTestBase
+    {
+        [Theory]
+        [MemberData(nameof(DnxSdks))]
+        public void GlobalJsonInProjectDir(DnxSdk sdk)
+        {
+            const string solutionName = "GlobalJsonInProjectDir";
+
+            var solution = TestUtils.GetSolution<DnuPublishTests>(sdk, solutionName);
+            var projectPath = Path.Combine(solution.RootPath, "Project");
+            var outputPath = Path.Combine(solution.RootPath, "Output");
+
+            var expectedOutputProjectJson = new JObject
```

Comment:
I'm checking for file content here instead of just existence to make sure that the values inside the original global.json is copied over. Unfortunately this makes the test very long since the contents of all files must be defined.